packages/@react-spectrum/s2/stories/ActionButton.stories.tsx

```diff
@@ -198,3 +201,19 @@ export const Avatars: Story = {
     );
   }
 };
+
+export const NotificationBadges: Story = {
+  render: (args) => {
+    let badgeValue = 10;
+    let formattedValue = new NumberFormatter('en-US').format(badgeValue);
+
+    return (
+      <div style={{display: 'flex', gap: 8, padding: 8, justifyContent: 'center'}}>
+        <ActionButton aria-label="Messages has new activity" {...args}><CommentIcon /><NotificationBadge /></ActionButton>
+        <ActionButton aria-label={`${formattedValue} notifications`} {...args}><BellIcon /><NotificationBadge value={badgeValue} /></ActionButton>
```

Comment:
are people okay with this story? tried to show how you might want to format the number when using it in an aria-label. the hard coding of the locale was because i ran into some storybook stuff and figured it would be easier this way.

Reply:
general idea, yes, but let's figure out why the hook didn't work, happy to look on monday

Reply:
oh it's a long story but basically, ran into linting issues with useLocale because it was inside the render function. so then i moved it outside of that but that messed up the "show code". tried to fix that but then it wasn't really happy with the `${formattedValue} notifications` part which is the exact part that i'm trying to show. i would have to change it to `{formattedValue} notifications` but since people might copy our code and expect it to work, that wouldn't be too great because it wouldn't work

Reply:
nvm...i've figured it out